src/components/images/index.js

```diff
@@ -0,0 +1,19 @@
+/**
+* Copyright 2012-2016, Plotly, Inc.
+* All rights reserved.
+*
+* This source code is licensed under the MIT license found in the
+* LICENSE file in the root directory of this source tree.
+*/
+
+'use strict';
+
+
+var draw = require('./draw');
+var supplyLayoutDefaults = require('./defaults');
+
+
+module.exports = {
+    draw: draw,
```

Comment:
you should attach the image attributes to `layoutAttributes` here and add `image': Image,` below this [line](https://github.com/plotly/plotly.js/blob/df5f58ed82c7f208da3058d5276466234f3eb2af/src/plots/layout_attributes.js#L207) so that `Plotly.PlotSchema.get()` includes the `layout.images` attribute in its output.